README_BUILD.md

```diff
@@ -93,8 +98,23 @@ When importing or creating custom types, follow these steps:
 6. The rake task will also copy the DSC resources into `lib/puppet_x/dsc_resources/`. This is necessary for the module to find the DSC resource implementations at runtime when applying the Puppet DSC resources.
 7. Enjoy!
 
-## Partial support for DSC composite resources.
-If you need a puppet type for a DSC composite resources, you have to define a xxx.schema.mof file next to your xxx.Schema.psm1 file.
+### Concise Steps to Build Custom DSC Resources
+
+```bash
+# perform an initial build to setup local copy
+bundle exec rake dsc:build
+# import your custom DSC Resource, repeat for each seperate directory
+bundle exec rake dsc:resources:import["path/to/your/types"]
+# parse the MOF files and build the types
+bundle exec rake dsc:types:build
+# commit the files to your fork
+git add . && git commit -m "new custom DSC Resource"
+```
+
+## Partial support for DSC composite resources
+
+If you need a puppet type for a DSC composite resources, you have to create a `xxx.schema.mof` file next to your `xxx.schema.psm1` file.
```

Comment:
I'm not 100% sure this works, and it's out of scope for this ticket. I will ticket this part as a spike/research/document so that this does not endanger the release